specs/waku/v2/waku-store.md

```diff
@@ -2,36 +2,88 @@
 title: Waku
 version: 2.0.0-beta1
 status: Draft
-authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>
+authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>, Sanaz Taheri <sanaz@status.im>
 ---
 
 # Table of Contents
 
 - [Abstract](#abstract)
+  * [Waku Store Protocol](#Waku-Store-Protocol)
```

Comment:
Usually this is autogenerated with markdown-toc and result would be lower-case-links. Easier to avoid mistakes and do less with this tool